src/reachy2_sdk/orbita/orbita.py

```diff
@@ -50,10 +53,10 @@ def __init__(
 
         self._compliant: bool
 
-        self._state: Dict[str, bool] = {}
-        self._register_needing_sync: List[str] = []
         self._joints: Dict[str, Any] = {}
+        self._reversed_joints: Dict[Any, str] = {}
```

Comment:
maybe a comment to explain what's reversed_joints is could be helpful. It is not to be understand as a mechanical joints that is reversed, but more some python code to easily retrieve the name of a joint.